scripts/set_instrument_list.py

```diff
@@ -92,7 +92,7 @@ def set_instlist(instruments_list, pv_address):
     if result != new_value:
```

Comment:
Same problem with compress_and_hex and dehex_and_decompress functions in this file. This is duplicate code so should just be imported from the other file, or the other way around.

Reply:
i'll just import the one from the server_common dir, that method is more documented